pkg/templater/yaml.go

```diff
@@ -18,3 +25,15 @@ func AppendYamlResources(resources ...[]byte) []byte {
 
 	return b
 }
+
+func ObjectsToYaml(objs ...runtime.Object) ([]byte, error) {
```

Comment:
whats the reason of doing that? to me it looks like part of the templater since we are generating yaml template. (call `AppendYamlResources` method). unless we wanna move both methods to the yamlutils

Reply:
Yeah I was thinking about both methods
Having the first was already a bit on the fence of not making sense
I think the second one almost guarantees having a separate package